docs/plugins/css/PostCSSPlugin.md

```diff
@@ -42,58 +47,129 @@ FuseBox.init({
 ```
 
 ### Require file in your code
+
 ```js
 import "./styles/main.css"
 ```
 
+### Plugins
+
+The `PostCSSPlugin` take PostCss plugins as a first argument:
+
+```js
+FuseBox.init({
+    plugins : [
+         [PostCSSPlugin([
+             require('postcss-import'),
+             // You can optionally pass options to the plugins
+             require('postcss-url')({url: "rebase"}),
+             require('postcss-nested')
+         ]), 
+         CSSPlugin()]
+    ]
+});
+```
+
+If you require more options, you can pass them as a second argument to the plugin:
+
+```js
+FuseBox.init({
+    plugins : [
+         [PostCSSPlugin(
+             // postcss plugins
+             [require('postcss-url')({url: "rebase"})], 
+             {
+                 // should fusebox generate sourcemaps (see below), default: true
+                 sourceMaps: false,
+                 // additional paths for css resolution (see below)
+                 paths: [],
+                 // all other options will go to the postcss process function (see below) 
+                 parser: parser,
+            }
+         ), CSSPlugin()]
+    ]
+});
+```
+
+Or like this:
+
+```js
+FuseBox.init({
+    plugins : [
+         [PostCSSPlugin({
+             // postcss plugins
+             plugins: [require('postcss-url')({url: "rebase"})],
```

Comment:
@nchanged Do you want to remove this to discourage the usage of this option ?

Reply:
Let's keep it please, we will remove it in the next release. Probably deprecation warning?